src/SFA.DAS.AssessorService.EpaoImporter/PrintProcessCommand.cs

```diff
@@ -135,49 +135,65 @@ private async Task UploadCertificateDetailsToPinter()
                 var batchLogResponse = await _assessorServiceApi.GetCurrentBatchLog();
 
                 var batchNumber = batchLogResponse.BatchNumber + 1;
-                var certificates = (await _assessorServiceApi.GetCertificatesToBePrinted()).ToList()
-                    .Sanitise(_aggregateLogger);
+                var certificates = (await _assessorServiceApi.GetCertificatesToBePrinted()).ToList();
```

Comment:
We're not Sanitising bad certificates any more?

Reply:
I have updated the code so that we are sanitzing before processing